### PR TITLE
Fix meetup build error

### PR DESCRIPTION
**Pull request by plaindocs**
I'm not really sure why the build started failing soon after I merged #463. Investigating. 

https://travis-ci.org/writethedocs/www/builds/543128866 

@mxsasha any idea?

I restarted the last successful build https://travis-ci.org/writethedocs/www/builds/542638416 and it now errors, so might be a dependency somewhere?

### CODE REVIEW

**Comment by mxsasha** [comment]
The traceback is a bit confusing, because the issue does not actually seem to be with the `docs/conf.py` file that `eval_config_file` is attempting to read. The code seems to be trying to decode an empty JSON response, and therefore my best guess is a flaky meetup API. This is consistent with the fact that it almost always fails locally for me, at a different point when getting meetup info. When I dump the response content by hacking `meetup/api.py`, I do see server response headers, with `Content-Length: 0` and `Content-Type: application/json;charset=utf-8`, which matches an entirely empty JSON response, which I think is indeed not valid.

In other words, best guess is an extremely flaky meetup API, with a possible hacky solution being to have `meetup_events.py:49` simply retry if a `JSONDecodeError` is raised. I can't assess immediately whether a single call is made per meetup or one for all meetups, because in the latter case it'll help poorly. Perhaps it's easy to call the API once per meetup instead, so that we can implement the retry in our own code. Or, there may be an entirely different solution.

**Comment by plaindocs** [comment]
I reckon you're right. Seems intermittent. 
I'll build some error protection in so failed meetups doesn't break the rest of the build.

**Comment by plaindocs** [comment]
Urgh https://travis-ci.org/writethedocs/www/jobs/544273361

**Comment by mxsasha** [comment]
> Urgh https://travis-ci.org/writethedocs/www/jobs/544273361

Wait, that's an entirely different issue, right?

**Comment by plaindocs** [comment]
I think we can just close this.
